File: app/src/main/java/com/kicktraineroverlay/MainActivity.java
package com.kicktraineroverlay;

import android.app.Activity;
import android.os.Bundle;
import android.provider.Settings;
import android.content.Intent;
import android.net.Uri;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        LinearLayout layout = new LinearLayout(this);
        layout.setOrientation(LinearLayout.VERTICAL);
        layout.setPadding(30, 30, 30, 30);

        TextView title = new TextView(this);
        title.setText("Kick Trainer Overlay\n\nابتدا اجازه نمایش روی برنامه‌ها را فعال کن.");
        title.setTextSize(18);

        Button permission = new Button(this);
        permission.setText("فعال کردن Overlay");

        permission.setOnClickListener(v -> {
            Intent intent = new Intent(
                    Settings.ACTION_MANAGE_OVERLAY_PERMISSION,
                    Uri.parse("package:" + getPackageName())
            );
            startActivity(intent);
        });

        layout.addView(title);
        layout.addView(permission);

        setContentView(layout);
    }
}

File: app/src/main/java/com/kicktraineroverlay/MainActivity.kt
package com.kicktraineroverlay

import android.app.Activity
import android.os.Bundle
import android.provider.Settings
import android.content.Intent
import android.net.Uri
import android.widget.Button
import android.widget.LinearLayout
import android.widget.TextView

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val layout = LinearLayout(this)
        layout.orientation = LinearLayout.VERTICAL
        layout.setPadding(30, 30, 30, 30)

        val title = TextView(this)
        title.text = "Kick Trainer Overlay\n\nابتدا اجازه نمایش روی برنامه‌ها را فعال کن."
        title.textSize = 18f

        val permission = Button(this)
        permission.text = "فعال کردن Overlay"

        permission.setOnClickListener {
            val intent = Intent(
                Settings.ACTION_MANAGE_OVERLAY_PERMISSION,
                Uri.parse("package:$packageName")
            )
            startActivity(intent)
        }

        layout.addView(title)
        layout.addView(permission)

        setContentView(layout)
    }
}
